README.md

```diff
@@ -6,27 +6,37 @@ This is a template that can be used to create Vapor applications that include Gr
 
 ## Example Queries:
```

Comment:
Could you please add an example query for the createTodo mutation with the userId?